### PR TITLE
bump bigdecimal, num-bigint and num-integer

### PR DESCRIPTION
**Pull request by leodasvacas**

### CODE REVIEW

**Comment by marioidival** [comment]
god

File: diesel/Cargo.toml [hunk 1]
```diff
@@ -25,10 +25,10 @@ time = { version = "0.1", optional = true }
 url = { version = "1.4.0", optional = true }
 uuid = { version = ">=0.2.0, <0.7.0", optional = true, features = ["use_std"] }
 ipnetwork = { version = "0.12.2", optional = true }
-num-bigint = { version = "0.1.41", optional = true }
+num-bigint = { version = "0.2", optional = true }
```
Comment (sgrif) [inline diesel/Cargo.toml]: This is a breaking change. Are we unable to support both versions?

File: diesel/Cargo.toml [hunk 2]
```diff
@@ -25,10 +25,10 @@ time = { version = "0.1", optional = true }
...
 num-traits = { version = "0.2", optional = true }
-num-integer = { version = "0.1.32", optional = true }
-bigdecimal = { version = ">= 0.0.10, < 0.0.13", optional = true }
+num-integer = { version = "0.1.39", optional = true }
```
[the remark this answers is not in the document]
Comment (sgrif) [reply diesel/Cargo.toml]: Assuming that we're able to support `0.1.32`, yes

File: diesel/Cargo.toml (hunk 1 above)
Comment (sgrif) [reply diesel/Cargo.toml]: `> 0.1, <= 0.3`

**Comment by leodasvacas** [comment]
@sgrif thanks for the review! Does it look correct now?

**Comment by weiznich** [comment]
Someone needs to fix the ci. (We will not merge a PR that breaks the ci :wink:)

> Is there any reason that these dependencies have an upper bound on version selection?

We need a upper bound there because otherwise those dependencies could break diesel if they break the functions that are used inside of diesel. Using only `> some version` would allow cargo to pull in a new possible incompatible major version.

**Comment by kestred** [comment]
@weiznich: please see https://github.com/diesel-rs/diesel/pull/1837 (tests finished successfully)

**Comment by sorccu** [comment]
This issue can be closed since #1837 is the exact same thing and it's been merged.

**Comment by weiznich** [comment]
As @sorccu pointed out this was already done by #1837.

**Pull request closed without merging**